Scalar.Installer.Mac/uninstall_scalar.sh

```diff
@@ -27,33 +27,37 @@ function UnInstallScalar()
     "org.scalar.service"
     )
     for nextLaunchAgent in "${launchAgents[@]}"; do
-    	for uid in $(ps -Ac -o uid,command | grep -iw "loginwindow" | awk '{print $1}'); do
-			isLoadedCmd="sudo launchctl kill SIGCONT gui/$uid/$nextLaunchAgent"
+        for uid in $(ps -Ac -o uid,command | grep -iw "loginwindow" | awk '{print $1}'); do
+            isLoadedCmd="sudo launchctl kill SIGCONT gui/$uid/$nextLaunchAgent"
             echo "$isLoadedCmd"
             if $isLoadedCmd; then
                 unloadCmd="launchctl bootout gui/$uid /Library/LaunchAgents/$nextLaunchAgent.plist"
                 echo "Unloading Service: '$unloadCmd'..."
-                eval $unloadCmd || exit 1                
+                error=$(eval $unloadCmd)
```

Comment:
This sets `error` to the `stdout` of `eval $unloadCmd`. I think what you want is this instead:

```sh
eval $unloadCmd
error=$?
```

Reply:
Ah yes

Reply:
but doesn't `eval` pass the exit code of the string you give it?

Reply:
At least for Bash:
https://www.gnu.org/software/bash/manual/html_node/Bourne-Shell-Builtins.html

> eval
> `eval [arguments]`
>The arguments are concatenated together into a single command, which is then read and executed, and its exit status returned as the exit status of eval. If there are no arguments or only empty arguments, the return status is zero.

I'll update to your suggestion however to avoid any doubt or differences between shells.

Reply:
Oh wait. never mind. I just re-read what you said. You are correct.

It's too hot today 🥵 

![too-hot](https://user-images.githubusercontent.com/5658207/85762558-c2444a00-b70b-11ea-861a-c26f00d80a4a.gif)